resources/views/individuals/edit/about-you.blade.php

```diff
@@ -12,78 +12,106 @@
                 {{ __('Step :current of :total', ['current' => request()->get('step') ?? 1, 'total' => $individual->isConnector() ? 5 : 4]) }}<br />
                 {{ __('About you') }}
             </h2>
+            <x-interpretation class="interpretation--start" name="{{ __('About you', [], 'en') }}" />
 
             <hr class="divider--thick">
 
-            <div class="field @error('name') field--error @enderror">
-                <x-hearth-label for="name"><x-required>{{ __('Name') }}</x-required></x-hearth-label>
-                <x-hearth-hint for="name">
-                    {{ __('This is the name that will be displayed on your page. This does not have to be your legal name.') }}
-                </x-hearth-hint>
-                <x-hearth-input name="name" type="text" :value="old('name', $individual->name)" required hinted />
-                <x-hearth-error for="name" />
-            </div>
+            <fieldset>
+                <div class="field @error('name') field--error @enderror">
+                    <x-hearth-label for="name"><x-required>{{ __('Name') }}</x-required></x-hearth-label>
+                    <x-hearth-hint for="name">
+                        {{ __('This is the name that will be displayed on your page. This does not have to be your legal name.') }}
+                    </x-hearth-hint>
+                    <x-interpretation class="interpretation--start" name="{{ __('Name (required)', [], 'en') }}" />
+                    <x-hearth-input name="name" type="text" :value="old('name', $individual->name)" required hinted />
+                    <x-hearth-error for="name" />
+                </div>
+            </fieldset>
+
+            <hr class="divider--thin">
 
             <fieldset>
                 <legend>{{ __('Where do you live?') }}</legend>
+                <x-interpretation class="interpretation--start" name="{{ __('Where do you live?', [], 'en') }}" />
 
                 <div class="field @error('region') field--error @enderror">
                     <x-hearth-label
                         for="region"><x-required>{{ __('Province or territory') }}</x-required></x-hearth-label>
+                    <x-interpretation class="interpretation--start"
+                        name="{{ __('Province or territory (required)', [], 'en') }}" namespace="province_territory" />
```

Comment:
Should remove "(required)" from the interpretation name. Also did we decide to update all the form inputs to have the "required" and "optional" in the namespace. I think that was the case, and we should update all of the form input related ones. Please correct me if I'm mistaken.